src/app/website/company-home/many-posibilities/many-posibilities.component.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../shared/_bits.scss';
+
+.manypossibility{
+  overflow-x:hidden; 
+  width:100%;
+}
+.manypossibility__content{
+  margin-left:4%;
+  margin-right:4%;
+  
+  @include breakpoint(phablet) {
+    margin-left:5%;
+    margin-right:5%;
+  }
+}
+.align {
```

Comment:
if a selector is not specific and this can be applied to other components then you will have to make it as a `sass placeholder or mixin` else you have to use `many-posisiblility__align`. Else it will affect other components if they have `align` in their selector